src/mlpack/methods/preprocess/preprocess_describe_main.cpp

```diff
@@ -0,0 +1,231 @@
+  for (size_t i = 0; i < input.n_elem; ++i)
+  {
+    sum += pow(input(i) - rowMean, Nth);
+  }
+  return sum;
+}
+/**
+ * Calculates Skewness of the given vector.
+ *
+ * @param input Vector that captures a dimension of a dataset
+ * @param rowStd Standard Deviation of the given vector.
+ * @param rowMean Mean of the given vector.
+ * @return Skewness of the given vector.
+ */
+double Skewness(const arma::rowvec& input,
+    const double& rowStd,
+    const double& rowMean,
+    const bool population)
+{
+  double skewness = 0;
+  double S3 = pow(rowStd, 3);
+  double M3 = SumNthPowerDeviations(input, rowMean, 3);
+  double n = input.n_elem;
+  if (population)
+  {
+    // Calculate Population Skewness
+    skewness = n * M3 / (n * n * S3);
+  }
+  else
+  {
+    // Calculate Sample Skewness
+    skewness = n * M3 / ((n-1) * (n-2) * S3);
+  }
+  return skewness;
+}
+/**
+ * Calculates kurtosis of the given vector.
+ *
+ * @param input Vector that captures a dimension of a dataset
+ * @param rowStd Standard Deviation of the given vector.
+ * @param rowMean Mean of the given vector.
+ * @return Kurtosis of the given vector.
+ */
+double Kurtosis(const arma::rowvec& input,
+    const double& rowStd,
+    const double& rowMean,
+    const bool population)
+{
+  double kurtosis = 0;
+  double M4 = SumNthPowerDeviations(input, rowMean, 4);
+  double n = input.n_elem;
+  if (population)
+  {
+    // Calculate Population Excess Kurtosis
+    double M2 = SumNthPowerDeviations(input, rowMean, 2);
+    kurtosis = n * (M4 / pow(M2, 2)) - 3;
+  }
+  else
+  {
+    // Calculate Sample Excess Kurtosis
+    double S4 = pow(rowStd, 4);
+    double norm3 = (3 * (n-1) * (n-1)) / ((n-2) * (n-3));
+    double normC = (n * (n+1))/((n-1) * (n-2) * (n-3));
+    double normM = M4 / S4;
+    kurtosis = normC * normM - norm3;
+  }
+  return kurtosis;
+}
+/**
+ * Calculates standard error of standard deviation.
+ *
+ * @param input Vector that captures a dimension of a dataset
+ * @param rowStd Standard Deviation of the given vector.
+ * @return Standard error of the stanrdard devation of the given vector.
+ */
+double StandardError(const arma::rowvec& input, const double rowStd)
+{
+   return rowStd / sqrt(input.n_elem);
+}
+
+int main(int argc, char** argv)
+{
+  // Parse command line options.
+  CLI::ParseCommandLine(argc, argv);
+  const string inputFile = CLI::GetParam<string>("input_file");
+  const size_t dimension = static_cast<size_t>(CLI::GetParam<int>("dimension"));
+  const size_t precision = static_cast<size_t>(CLI::GetParam<int>("precision"));
+  const size_t width = static_cast<size_t>(CLI::GetParam<int>("width"));
+  const bool population = CLI::HasParam("population");
+
+  // Load the data
+  arma::mat data;
+  data::Load(inputFile, data, false, true /*transpose*/);
+
+  // Generate boost format recipe.
+  const string widthPrecision("%-"+
+    to_string(width)+ "." +
+    to_string(precision));
+  const string widthOnly("%-"+
+    to_string(width)+ ".");
```

Comment:
Be sure to follow the style guide---wrapped lines should be indented twice (not once), and try to place spaces between operators (like `a + b` not `a+b`) for readability. :)